src/addon.cpp

```diff
@@ -1153,8 +1153,12 @@ ADDON_STATUS addon::Create(void)
       }
 
       // Load the device settings
+#ifdef USB_DEVICE_SUPPORT
       m_settings.device_connection =
           kodi::addon::GetSettingEnum("device_connection", device_connection::usb);
+#else
+      m_settings.device_connection = device_connection::rtltcp;
+#endif
       m_settings.device_connection_usb_index =
           kodi::addon::GetSettingInt("device_connection_usb_index", 0);
```

Comment:
You only need to read this property value if `USB_DEVICE_SUPPORT`is defined.

Reply:
Have changed there, also on the SettingsChange function is now the `USB_DEVICE_SUPPORT`